index.css

```diff
@@ -0,0 +1,158 @@
+@import url(https://fonts.googleapis.com/css?family=Playfair+Display|Poppins);
+
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  width: 100vw;
+  overflow-x: hidden;
+}
+

```

Comment:
- [ ] The page it has a width that is more  than the [figma template has](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)